pbspro.spec

```diff
@@ -308,6 +308,7 @@ cd build
 %make_install
 mandir=$(find %{buildroot} -type d -name man)
 [ -d "$mandir" ] && find $mandir -type f -exec gzip -9 -n {} \;
+install -D %{buildroot}/%{pbs_prefix}/libexec/pbs_init.d %{buildroot}/etc/init.d/pbs
```

Comment:
The client rpm does not have pbs_init.d, and will show a warning when it hits this line.

Reply:
I tested installing client package and there was not warning. The %install section happens during RPM build not during package install. And because`pbs_init.d` is excluded from the client package, RPM will not install it.